blocks/easy-forms-block/easy-forms-block.php

```diff
@@ -109,3 +109,16 @@ public function render_block( $attributes, $content ) {
 		);
 	}
 }
+
+function my_mario_block_category( $categories, $post ) {
+	return array_merge(
+		$categories,
+		array(
+			array(
+				'slug' => 'easy-forms',
+				'title' => __( 'Easy Forms', 'easy-forms' ),
+			),
+		)
+	);
+}
+add_filter( 'block_categories', 'easy_form_category', 10, 2);
```

Comment:
This is added to the constructor then except you need to reference the class's method instead of a function. Do that by passing an array `array( $this, 'easy_forms_register_category' );` Also, name your method `easy_forms_register_category`.